app/views/collections/show.html.slim

```diff
@@ -89,17 +89,26 @@
               .input-group
                 = text_field_tag 'user', nil, form: 'share', class: 'form-control'
                 = submit_tag t('.button.share'), form: 'share', class: 'btn btn-light'
-          .dropup.btn-group
+          .dropdown.btn-group
+            - if @collection.tasks.empty?
+              div data-bs-toggle='tooltip' title=t('.download_no_exercises') data-bs-delay=150
+                = button_tag class: 'btn btn-important dropdown-toggle disabled', 'data-bs-toggle': 'dropdown' do
+                  => t('common.button.download_zip')
```

Comment:
The disabled button doesn't wrap correctly in three lines if needed (but the enabled button does):

**Disabled:**
![Bildschirmfoto 2024-08-13 um 19 10 27](https://github.com/user-attachments/assets/74f0bbcd-8ff7-4c9e-86d4-58bcaf605f3d)

**Enabled:**
![Bildschirmfoto 2024-08-13 um 19 11 12](https://github.com/user-attachments/assets/713783c7-e4ae-4fd3-b703-daeb71b6fffc)